src/main/java/com/movierecommender/project/MovieController.java

```diff
@@ -77,13 +75,17 @@ public String index() {
     }
 
     @GetMapping("/login")
-    public String login() {return "login";}
+    public String login() {
+        return "login";
+    }
 
     @GetMapping("/signup")
     public String signup() {
         return "signup";
     }
 
-    @GetMapping("/Profile")
-    public String Profile(){ return "profile"; }
+    @GetMapping("/user-profile")
+    public String userProfile() {
+        return "profile";
+    }
```

Comment:
I did some changes in GetMapping /profile, this mapping was redundant because it served the same view ("profile") as the /user-profile mapping. Reference: https://www.thecodebuzz.com/restful-api-url-naming-conventions-best-practices/